.github/workflows/release-update-stable-tag.yml

```diff
@@ -66,6 +69,41 @@ jobs:
         with:
           fetch-depth: 0
 
+      - name: Get trunk version
+        id: get-trunk-version
+        run: |
+          TRUNK_VERSION=$(awk '/Version:/ {print $3}' plugins/woocommerce/woocommerce.php | sed -E 's/^([0-9]+\.[0-9]+)\.[0-9]+.*/\1/' | sed 's/-.*//')
+          echo "Trunk version is $TRUNK_VERSION"
+          echo "trunk-version=$TRUNK_VERSION" >> $GITHUB_OUTPUT
+
+      - name: Get target branches for stable tag update
+        id: get-branches
+        uses: actions/github-script@v7
+        with:
+          script: |
+            const targetVersion = '${{ steps.extract-version.outputs.version }}';
+            const trunkVersion = '${{ steps.get-trunk-version.outputs.trunk-version }}';
+
+            console.log(`Target version: ${targetVersion}`);
+            console.log(`Trunk version: ${trunkVersion}`);
+
+            const [targetMajor, targetMinor] = targetVersion.split('.').map(Number);
+            const [trunkMajor, trunkMinor] = trunkVersion.split('.').map(Number);
+
+            const branches = ['trunk'];
+
+            for (let major = targetMajor; major <= trunkMajor; major++) {
+              const startMinor = (major === targetMajor) ? targetMinor : 0;
+              const endMinor = (major === trunkMajor) ? trunkMinor : 9;
+
+              for (let minor = startMinor; minor <= endMinor; minor++) {
+                branches.push(`release/${major}.${minor}`);
+              }
+            }
```

Comment:
This is including the version in `trunk` (right now, this would be `release/10.1`) which is a branch that will never exist. 

Also, given the version numbers are only `0.1` apart, maybe we can simplify the code to something along these lines? 👇 

```js
const branches = ['trunk'];
for (let v = Number(`${targetMajor}.${targetMinor}`); v < Number(`${trunkMajor}.${trunkMinor}`); v += 0.1) {
	branches.push(`release/${v.toFixed(1)}`);
}
```

Do you see any problem with something like that? Maybe it's not worth it. Let me know what you think.